### ISSUE TITLE
[Translation][Website] "made edits to title or description of this studio" etc. need place holders

### ISSUE THREAD

**Issue by gunchleoc** [LLK/scratch-flash#697]
This string:

```
made edits to title or description of this studio
```

_templates/galleries/includes/activity_list.html:46_

will be assembled by django to:

```
GunChleoc made edits to title or description of this studio 12:09 p.m.
```

but my language needs the following word order:

```
made GunChleoc edits to title or description of studio this 12:09 p.m.
```

because the verb always comes first.

All activity update strings have this problem. I recommend using place holders:

```
%(username)s made edits to title or description of this studio %(time)s
```

In this example, I have added a place holder for the time too, in case other languages need this. If the place holders could be reversible, this would safeguard against all eventualities.

Long story short:
- When assembling sentences from multiple parts, always use place holders in your string design.
- When there are multiple place holders in the same string, make them reversible.

I'd be surprised if this has never come up before, because you have so many translations, a lot of which will be struggling with this kind of thing.

Also reported on the forums: https://scratch.mit.edu/discuss/topic/102364/

**Comment by 2jour** [comment]
Created a sub issue

**Comment by gunchleoc** [comment]
The new strings that have hit Pootle recently are even worse.

For example:

    was promoted to manager by 

Results in:

    X was promoted to manager by  Y

But I need:

    Y promoted X to manager

There is also this snippet:

    from studio 

Which really shouldn't be hard coded.

I don't know how this is implemented, but would it be possible to add new strings correctly, even if you don't have time right now to fix the old strings? I'm starting to feel like a contortionist :'(
